katharsis-client/src/main/java/io/katharsis/client/KatharsisClient.java

```diff
@@ -461,4 +448,8 @@ public HttpAdapter getHttpAdapter() {
 	public ModuleRegistry getModuleRegistry() {
 		return moduleRegistry;
 	}
+
+	public ClientDocumentMapper getDocumentMapper() {
```

Comment:
we have to discuss that at some poitn in the forum. But lombok is not really an option from my side. It makes the code unreadable/discoverable (finding references, etc.) and messes up the Eclipse IDE. But I agree we need something to avoid some boiler plate (hashCode/equals) and maybe takeover the generation of getters/setters of POJOs. https://immutables.github.io/ might be something.